azurerm/resource_arm_application_gateway.go

```diff
@@ -1305,13 +1326,16 @@ func flattenApplicationGatewayBackendHTTPSettings(input *[]network.ApplicationGa
 			if path := props.Path; path != nil {
 				output["path"] = *path
 			}
+			output["connection_draining"] = flattenApplicationGatewayConnectionDraining(props.ConnectionDraining)
+
 			if port := props.Port; port != nil {
 				output["port"] = int(*port)
 			}
+
 			if pickHostNameFromBackendAddress := props.PickHostNameFromBackendAddress; pickHostNameFromBackendAddress != nil {
 				output["pick_host_name_from_backend_address"] = *pickHostNameFromBackendAddress
 			}
-			output["protocol"] = string(props.Protocol)
```

Comment:
I'm going to push a commit to add this line back in which should fix the test failure - I hope you don't mind 👍